kentikapi/internal/resources/alerting_test.go

```diff
@@ -27,7 +27,7 @@ func TestCrerateManualMitigation(t *testing.T) {
 	expectedRequestBody := `{"ipCidr":"192.168.0.0/24","platformID":"1234","methodID":"12345","minutesBeforeAutoStop":"20"}`
 
 	transport := &api_connection.StubTransport{ResponseBody: createResponsePayload}
-	alertingAPI := resources.NewAlertingAPI(transport)
+	alertingAPI := resources.NewAlertingAPI(transport, false)
```

Comment:
```suggestion
	alertingAPI := resources.NewAlertingAPI(transport, true)
```
I suggest to enable logging payloads in tests to:
- verify this code path works
- help with possible debugging of tests

Reply:
Done